temporal-sdk/src/main/java/io/temporal/internal/worker/LocalActivityWorker.java

```diff
@@ -220,7 +220,8 @@ public Throwable wrapFailure(Task task, Throwable failure) {
       metricsScope.counter(MetricsType.LOCAL_ACTIVITY_TOTAL_COUNTER).inc(1);
 
       Stopwatch sw = metricsScope.timer(MetricsType.LOCAL_ACTIVITY_EXECUTION_LATENCY).start();
-      ActivityTaskHandler.Result result = handler.handle(activityTask.build(), metricsScope, true);
+      ActivityTaskHandler.Result result =
+          handler.handle(new ActivityTask(activityTask.build(), () -> {}), metricsScope, true);
```

Comment:
nit: Might be worth having a constructor overload for empty completion handles

Reply:
I don't think this warrants a separate constructor. Would prefer to keep as is.